src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingService.cs

```diff
@@ -93,8 +93,7 @@ internal static LanguageService LanguageServiceInstance
         /// <param name="context"></param>
         public void InitializeService(ServiceHost serviceHost)
         {
-            serviceHost.SetRequestHandler(ScriptingScriptAsRequest.Type, HandleScriptingScriptAsRequest);
```

Comment:
This is still needed by VSCode-MSSQL, please do not remove as it's a breaking change. Given the other discussion points we should consider at  minimum adding a timeout to the ScriptAsRequest so that it does timeout like before, or adding a new / updated method that returns immediately with a "Script Queued" result and then notifies when scripting is complete or failed.

Otherwise we're going to hold up the entire pipeline and no other queries will be processable in this time.

Reply:
VSCode-MSSQL doesn't actually use this request type, so I'm letting it be removed.